pkg/operations/cordondrainvm.go

```diff
@@ -200,20 +208,23 @@ func (o *drainOperation) evictPods(pods []v1.Pod, policyGroupVersion string) err
 			} else {
 				errCh <- errors.Wrapf(err, "error when waiting for pod %q terminating", pod.Name)
 			}
-		}(pod, doneCh, errCh)
+		}(ctx, pod, doneCh, errCh)
 	}
 
 	doneCount := 0
 	for {
 		select {
 		case err := <-errCh:
+			cancel()
```

Comment:
Perhaps, put the cancel in a defer outside of the for loop.

Reply:
Done. Thanks!